### ISSUE TITLE
Dropdown with Hover doesn't work on tablet devices. 

### ISSUE THREAD

**Issue by melineWb** [Dogfalo/materialize#6616]
$('.dropdown-button').dropdown({hover: true}) doesn't work onClick on Tablet devices. 
Tested on Android

**Comment by doughballs** [comment]
Yes - hover doesn't work on touch devices. Only use `hover:true` if you're sure the device isn't a touch device.

https://stackoverflow.com/questions/4817029/whats-the-best-way-to-detect-a-touch-screen-device-using-javascript
